src/main/java/bio/terra/cli/businessobject/Resource.java

```diff
@@ -54,6 +55,7 @@
    */
   public enum Type {
     GCS_BUCKET,
+    GCS_BUCKET_FILE,
```

Comment:
I don't see a lot of context in the Jira ticket (maybe @gjuggler could add color), and I'm not sure if the use of "file" to describe a GCS object is already pervasive across Terra, but...

Hopefully not too pedantic, but I'd lean towards `GCS_OBJECT`/`GcsObject`, since GCS is object storage.  This might be a useful distinction if we ever support actual file storage like [filestore](https://cloud.google.com/filestore), where filesystem semantics are meaningful.

If others think that this is worth  doing, I'd also recommend `filePath` -> `objectName` elsewhere (IME the distinction between object name prefixes and real file system paths is meaningful and can cause confusion for end users).

Reply:
Actually just noticed that the WSM side is already implemented, so maybe this ship has sailed; feel free to disregard.

Reply:
Object instead of file sounds good to me.

The enum here is CLI-specific, so no need to tie it to WSM naming. One of the main reasons for "redefining" enums here that map to existing WSM client library enums was to let the CLI control display, and I think this fits that case.